depthai_demo.py

```diff
@@ -112,46 +130,69 @@ def print_sys_info(info):
     fps = FPSHandler() if conf.useCamera else FPSHandler(cap)
 
     if conf.useCamera or conf.args.sync:
-        pv = PreviewManager(fps, display=conf.args.show, colorMap=conf.getColorMap(), disp_multiplier=disp_multiplier)
+        pv = PreviewManager(fps, display=conf.args.show, colorMap=conf.getColorMap(), disp_multiplier=disp_multiplier, mouseTracker=True)
 
-        if conf.args.camera == "left" or conf.useDepth:
+        if conf.leftCameraEnabled:
             pm.create_left_cam(mono_res, conf.args.mono_fps)
-        if conf.args.camera == "right" or conf.useDepth:
+        if conf.rightCameraEnabled:
             pm.create_right_cam(mono_res, conf.args.mono_fps)
-        if conf.args.camera == "color":
-            pm.create_color_cam(rgb_res, conf.args.rgb_fps, conf.args.full_fov_nn, conf.useHQ)
+        if conf.rgbCameraEnabled:
+            pm.create_color_cam(nn_manager.input_size if conf.useNN else (300, 300), rgb_res, conf.args.rgb_fps, conf.args.full_fov_nn, conf.useHQ)
```

Comment:
for these king of hardcoded constansts would be better using a global const, so it's more transparent, easier to change, not hidden in the middle of the code somewhere.
300x300 is default in case NN doesn't specify input size? Or it's if NN was not requested? If latter then it would be better setting it to the same aspect ratio as the camera, e.g. 576x324, so it covers the full FOV

Reply:
It's when the nn is disabled and no input size was specified. I'll adjust it and store as a variable / param